.circleci/config.yml

```diff
@@ -27,14 +27,14 @@ defaults_Dependencies: &defaults_Dependencies |
   npm install -g node-gyp
 
 defaults_awsCliDependencies: &defaults_awsCliDependencies |
-    apk upgrade --no-cache
-    apk --no-cache add \
-            python3 \
-            py3-pip \
-            groff \
-            less \
-            mailcap
-    pip3 install --upgrade pip awscli==1.14.5 s3cmd==2.0.1 python-magic
+  apk upgrade --no-cache
+  apk --no-cache add \
+          python3 \
+          py3-pip \
+          groff \
+          less \
+          mailcap
+  pip3 install --upgrade pip awscli==1.14.5 s3cmd==2.0.1 python-magic
```

Comment:
I would suggest checking that`checkout` steps are after installing dependencies so checkout is done with the latest git